File: Java/Car.java
package Java;

class Car {
    Integer id;
    String license;
    Account driver;
    protected Integer passenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }   

    void printDataCar() {
        if(passenger != null) {
            System.out.println("License: " + license + " Driver name: " + driver.name + " Passengers: " + passenger);
        }
    }

    public Integer getPassenger(){
        return passenger;
    }

    public void setPassenger(Integer passenger){
        if (passenger == 4) {
            this.passenger = passenger;
        }else{
            System.out.println("Necesitas asignar 4 pasajeros");
        }  
    }
}

File: PHP/Car.php
<?php
require_once("account.php");
class Car {
    public $id;
    public $license;
    public $driver;
    protected $passenger;

    function __construct($license, $driver){
        $this->license = $license;
        $this->driver = $driver;
    }

    public function PrintDataCar(){
        if ($this->passenger != null){
            echo "Licencia: $this->license Driver: {$this->driver->name} <br>";
        }
    }

    public function getPassenger() {
        return $this->passenger;
    }

    public function setPassenger($passenger) {
        if ($passenger == 4){
            $this->passenger = $passenger;
        }
        else {
            echo "necesitas asignar 4 pasajeros";
        }
    }
}
?>